content/en/docs/notebooks/troubleshoot.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Troubleshooting Guide"
+title = "Troubleshooting guide for Kubeflow notebooks"
```

Comment:
Is Kubeflow notebooks a thing? 

This may be confusing. Are we talking about Jupyter notebooks only? Or other notebook types we have available in the Kubeflow dashboard?

Reply:
Good point @RFMVasconcelos. I borrowed the idea from the old description.

Check out the new suggested changes:

```
title = "Troubleshooting guide for Jupyter notebooks on Kubeflow"
description = "Fixing common problems of Jupyter notebook deployments on Kubeflow""
```

Reply:
Thank you for the suggestion @8bitmp3 ! Looking good!